src/components/Card.jsx

```diff
@@ -0,0 +1,272 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classnames from "classnames";
+import React from "react";
+import styled from "styled-components";
+import { faBolt } from "@fortawesome/free-solid-svg-icons";
+import { faTag } from "@fortawesome/free-solid-svg-icons";
+
+const Container = styled.div`
```

Comment:
I just pasted the HTML and CSS from the OwnedCardContent.vue from the other app. no need to overcomplicate it for now